build.sbt

```diff
@@ -1,7 +1,7 @@
 lazy val alpakka = project
   .in(file("."))
   .enablePlugins(PublishUnidoc)
-  .aggregate(amqp, cassandra, dynamodb, files, ftp, hbase, jms, mqtt, s3, simpleCodecs, sqs, sse)
+  .aggregate(amqp, cassandra, dynamodb, files, ftp, hbase, jms, mqtt, s3, simpleCodecs, sqs, sse, awslambda)
```

Comment:
I had trouble adding `docs` to the aggregate after the rebase. Don't know why it worked before rebase but not after? 

```
/Users/patrik/dev/alpakka/build.sbt:4: error: recursive value docs needs type
  .aggregate(amqp, cassandra, dynamodb, docs, files, ftp, hbase, jms, mqtt, s3, simpleCodecs, sqs, sse, awslambda)
```